webpack/ForemanTasks/Components/TasksTable/TasksTablePage.js

```diff
@@ -40,16 +40,20 @@ const TasksTablePage = ({ getBreadcrumbs, history, ...props }) => {
     }));
   };
 
-  const TaskSelectedAction = (id, name) => {
-    props.actionSelected(id, name, url);
+  const TaskSelectedAction = {
```

Comment:
Prefer using regular snake case (`taskSelectedAction`) unless it's a React component.

Reply:
Nice catch thanks